packages/components/src/Chip/Chip.css

```diff
@@ -0,0 +1,96 @@
+:root {
+  --chip-height: 40px;
+  --chip-radius: 20px;
+}
+
+.chip {
+  display: flex;
+  position: relative;
+  height: var(--chip-height);
+  padding: var(--space-small) var(--base-unit);
+  border: none;
+  border-radius: var(--chip-radius);
+  text-align: left;
+  cursor: pointer;
+  transition: all ease-in-out 0.12s;
+  align-items: center;
+}
+
+.chip:focus {
+  box-shadow: var(--shadow-focus);
+}
+
+.chip .prefix {
+  margin-left: calc(-1 * var(--space-small));
+  margin-right: var(--space-small);
+}
+
+.chip .suffix {
+  margin-left: var(--space-small);
+  margin-right: calc(-1 * var(--space-small));
+}
+
+.chip .prefix,
+.chip .suffix {
+  display: flex;
+  width: var(--space-large);
+  height: var(--space-large);
+  border-radius: var(--radius-circle);
+  background-color: var(--color-white);
+  align-items: center;
+  justify-content: center;
+}
+
+.chip .prefix.empty,
+.chip .suffix.empty {
+  display: none;
+}
+
+.chip .chipBar {
+  width: var(--space-minuscule);
+  height: var(--base-unit);
+  margin: 0 var(--space-small);
+  background-color: var(--color-greyBlue--light);
+}
+
+.chip .selected {
+  border-radius: var(--radius-circle);
+  background-color: var(--color-white);
+}
+
+.base {
+  color: var(--color-blue);
+  background-color: var(--color-greyBlue--lightest);
+}
+.base:hover,
+.base:focus {
+  background-color: var(--color-greyBlue--lighter);
+}
+
+.invalid,
+.invalid:hover,
+.invalid:focus,
+.subtle.invalid,
+.subtle.invalid:hover,
+.subtle.invalid:focus {
+  border: var(--border-base) solid var(--color-red);
```

Comment:
```suggestion
  border: var(--border-base) solid var(--color-critical);
```

Reply:
I have fixed this (I knew I was gonna get something wrong with the colours, I searched for rgb values in the code and found the root colours, but not the semantic colours). :)